src/NextGen/DonationForm/Blocks/DonationFormBlock/Block.php

```diff
@@ -103,33 +101,13 @@ private function createForm($attributes): Form
 
         $donationForm = new Form('DonationForm');
 
-        $donationForm->append(
-            Section::make('donationDetails')
-                ->label(__('Donation Details', 'give'))
-                ->append(
-                    Text::make('amount')
-                        ->label(__('Donation Amount', 'give'))
-                        ->defaultValue(50)
-                        ->required()
-                ),
-
-            Section::make('donorDetails')
-                ->label(__('Donor Details', 'give'))
-                ->append(
-                    Text::make('firstName')
-                        ->label(__('First Name', 'give'))
-                        ->required(),
-
-                    Text::make('lastName')
-                        ->label(__('Last Name', 'give'))
-                        ->required(),
-
-                    Email::make('email')
-                        ->label(__('Email', 'give'))
-                        ->required()
-                        ->emailTag('email')
-                ),
+        $formBlockData = json_decode(get_post($attributes['formId'])->post_content, false);
 
+        foreach( $formBlockData as $block ) {
+            $donationForm->append($this->convertFormBlockDataToFieldsAPI($block));
+        }
```

Comment:
This is causing nested `Section`'s.  Can you tweak this to use a different node type when inside this innerBlocks loop? I think `Group` might make the most sense.

Reply:
https://github.com/impress-org/givewp-next-gen/pull/6/commits/1c265f5740855f216237a4cfbf2b215249bb1bee

Reply:
@jonwaldstein I'm going to circle back to this one, see [b5f6ed6](https://github.com/impress-org/givewp-next-gen/pull/6/commits/b5f6ed6be044a36650bd1b99d7ccb4f472d79b92).